Xamarin/DrawXShared/RealmDraw.cs

```diff
@@ -338,7 +338,9 @@ private void DrawAPath(SKCanvas canvas, SKPaint paint, DrawPath drawPath)
         {
             using (var path = new SKPath())
             {
-                var pathColor = SwatchColor.ColorsByName[drawPath.color].Color;
+                // nasty hack because the Cocoa version defaults to Black
+                var pathColorName = (drawPath.color == "Black") ? "Charcoal" : drawPath.color;
```

Comment:
It has to map to one of the selected pencils rather than an arbitrary colour value. Anyway, moot point as the model is highly unlikely to change.